test/specs/manifestBuilder.e2e.ts

```diff
@@ -120,11 +121,11 @@ describe('Manifest Builder', async () => {
     const workbench = await browser.getWorkbench();
     await utilities.getTextEditor(workbench, 'manifest.xml');
     // Clear output before running the command
-    await utilities.runCommandFromCommandPrompt(workbench, 'View: Clear Output', 1);
+    await utilities.clearOutputView(Duration.seconds(2));
     await utilities.runCommandFromCommandPrompt(
       workbench,
       'SFDX: Retrieve Source in Manifest from Org',
-      1
+      Duration.seconds(2)
```

Comment:
Why was this changed from 1 second to 2 seconds?  Is there a flapper that happens when it's 1 second?

Reply:
here too